sdk/eventhub/test-resources.bicep

```
@@ -32,6 +32,7 @@ resource eventHubNamespace 'Microsoft.EventHub/namespaces@2024-01-01' = {
     zoneRedundant: false
     isAutoInflateEnabled: false
     maximumThroughputUnits: 0
+    disableLocalAuth: !supportsSafeSecretStandard
```

Sorry I got the condition wrong: https://github.com/Azure/azure-sdk-tools/pull/9781. This should be flipped.

```suggestion
    disableLocalAuth: supportsSafeSecretStandard
```